logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/LoginUseCase.kt

```diff
@@ -60,19 +65,24 @@ internal class LoginUseCaseImpl(
             }
 
             else -> return AuthenticationResult.Failure.InvalidUserIdentifier
+        }.flatMap { loginResult ->
+            serverConfigRepository.configByLinks(serverLinks)
+                .map { AuthenticationResult.Success(loginResult.first, loginResult.second, it.id) }
```

Comment:
```suggestion
        }.flatMap { (authData, ssoID) ->  ->
            serverConfigRepository.configByLinks(serverLinks)
                .map { AuthenticationResult.Success(authData, ssoID, it.id) }
```

Maybe we could use a deconstructing assignment here, to avoid the `.first` and `.second` calls, just to make it self-explanatory :D